src/pybind/mgr/ansible/module.py

```diff
@@ -70,20 +74,9 @@ def __init__(self, client, playbook, logger, result_pattern,
         # Logger
         self.log = logger
 
-        # An aditional filter of result events based in the event
-        self.event_filter = ""
-
         # Function assigned dinamically to process the result
         self.process_output = None
 
-        # Playbook execution object
-        self.pb_execution = PlayBookExecution(client,
-                                              playbook,
-                                              logger,
-                                              result_pattern,
-                                              params,
-                                              querystr_dict)
-
     def __str__(self):
          return "Playbook {playbook_name}".format(playbook_name = self.playbook)
```

Comment:
`AnsibleReadOperation` has no `playbook` property. Sorry that I missed that in the other PR.

Reply:
Removed. Now PlaybookOperation and ARSChangeOperation have their own __str__ method